client/src/views/Notifications/Components/Followers.vue

```diff
@@ -0,0 +1,71 @@
+<template>
```

Comment:
You have a similar component - Comments.vue, with only follow/unfollow and video-preview difference, please create one component to cover similar logic and difference you could handle with condition statements or via slots - https://vuejs.org/v2/guide/components-slots.html .